check-result.js

```diff
@@ -1,18 +1,14 @@
 function checkResult(player, computer) {
     if (player === computer) {
         return 'draw';
-    } else if (player === 'rock' && computer === 'paper') {
+    } else if ((player === 'rock' && computer === 'paper') || 
+    (player === 'paper' && computer === 'scissors') ||
+    (player === 'scissors' && computer === 'rock')) {
         return 'lose';
-    } else if (player === 'rock' && computer === 'scissors') {
+    } else if ((player === 'rock' && computer === 'scissors') || 
+    (player === 'paper' && computer === 'rock') || 
+    (player === 'scissors' && computer === 'paper')) {
```

Comment:
great job! you could take this a step further and hit it like this!

if (player === computer) {
        return 'draw';
} else if ((player === 'rock' && computer === 'scissors') || 
    (player === 'paper' && computer === 'rock') || 
    (player === 'scissors' && computer === 'paper')) {
        return 'win';
} else {
        return 'lose';
}